terraform/deployments/tfc-configuration/govuk-app.tf

```diff
@@ -0,0 +1,33 @@
+module "govuk-app-integration" {
+  source  = "alexbasista/workspacer/tfe"
+  version = "0.10.0"
+
+  project_name = "govuk-mobile-backend"
+  vcs_repo = {
+    identifier     = "alphagov/govuk-mobile-backend"
+    branch         = "main"
+    oauth_token_id = data.tfe_oauth_client.github.oauth_token_id
+  }
```

Comment:
Assuming that the terraform is living in this repo, instead of govuk-infrastructure.